subsys/bluetooth/controller/ll_sw/ull_master.c

```diff
@@ -405,10 +404,17 @@ uint8_t ll_connect_disable(void **rx)
 
 		cc->hdr.type = NODE_RX_TYPE_CONNECTION;
 		cc->hdr.handle = 0xffff;
+
+		/* NOTE: struct llcp_terminate.node_rx has uint8_t member
+		 *       following the struct node_rx_hdr to store the reason.
+		 */
 		*((uint8_t *)cc->pdu) = BT_HCI_ERR_UNKNOWN_CONN_ID;
```

Comment:
That's quite ugly and should be cleaned up. However, that should happen in a different PR (documenting ugliness should not be "rewarded" with scope creep).